loleaflet/src/control/Control.Menubar.js

```diff
@@ -625,7 +625,7 @@ L.Control.Menubar = L.Control.extend({
 				{uno: '.uno:FormatLine'},
 				{uno: '.uno:FormatArea'}
 			]},
-			{name: _UNO('.uno:SheetMenu', 'spreadsheet'), type: 'menu', menu: [
+			{name: _UNO('.uno:SheetMenu', 'spreadsheet'), id: 'sheet', type: 'menu', menu: [
```

Comment:
wouldn't it be better to use `sheetmenu` as id to avoid confusion?

Reply:
if you see the ID in DOM it would be menu-sheet

Reply:
okay then go ahead.